scylla/src/transport/load_balancing/default.rs

```diff
@@ -353,6 +405,10 @@ impl DefaultPolicy {
         node.is_enabled()
     }
 
+    fn rack_predicate(node: &NodeRef<'_>, preferred_rack: &Option<String>) -> bool {
```

Comment:
The other predicates are named after the condition they check, such as `is_alive` which verifies if a node is alive (it doesn't at the moment but will be;))). It may be worthwhile to rename `rack_predicate` to adhere to this convention, e.g. `is_rack_local`.